### DISCUSSION TITLE
Are there options for configuring wallpapers?

### DISCUSSION THREAD

**Discussion by ranenvious** [Almamu/linux-wallpaperengine#110] [Q&A]
Looking through the brief description on how to use it, it doesn't mention if there is a way to configure the wallpapers that you use. For many wallpapers this isn't an issue, but for some like the Audio Scratchboard wallpaper, it's basically a necessity. Is this functionality implemented or planned?

**Comment by Almamu** [comment]
It is planned, but no work has been done on it mainly because all those backgrounds that have options usually don't work in the software yet.

**Reply by Almamu** [reply]
@Churchexe I've added some support for these in the last few commits and you should be able to make changes to wallpaper's configuration as explained in the readme. My test have been successful, but testing on more backgrounds would be ideal!

**Reply by Almamu** [reply]
Take in mind that not all the parameters can be configured tho, I've added color, visibility, alpha and shader values, but there might be others parameters that might be missing from the list.

**Reply by Squidzy-code** [reply]
I know its harsh to ask for but a way to add the playlists so that it automatically swaps would be AMAZING.
